### PR TITLE
IO tesztek

### PR DESCRIPTION
**Pull request by nemethdav**
@hegyhati légyszíves nézz rá, hogy jók lesznek-e így az IO tesztek

### CODE REVIEW

**Review by hegyhati — CHANGES_REQUESTED** [review]
Rendben van alapvetoen, de a teszteke nagyon "egyhanguak". Ilyenkor azert jo volna ramenni edge case-ekre. 

Megnezni, mi van akkor,, ha nagyon nagyok a szamok. Vagy epp hogy pl 0 a tamadasa valamelyiknek. Mi van, ha mindkettonek 0?  Ilyenekre erdemes feldobni 1-1 test case-t es eldonteni, hogy mi a vart kimenet. Szoval meg 1-2 ilyet dobjatok hozza, de amugy jo lesz.

**Comment by nemethdav** [comment]
@hegyhati légyszíves nézd meg jók-e így a tesztek, valamint javítottam a Main.java-n is, hogy a megfelelő kimenetet kapjuk 0 és valamilyen betű(k) bevitelének esetén is.

File: tests/test11.txt.out [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Első egység HP-ja: Első egység DMG-je: Második egység HP-ja: Második egység DMG-je: a
+Nem érvényes értéket adott meg, adjon meg egy számot!
```
Comment (hegyhati) [inline tests/test11.txt.out:2]: Kerdes, hogy mennyire jo, hogy ez a fura dolog az expected output :-) De most elmegy.

**Review by hegyhati — APPROVED** [review]
